libs/barista-components/table/src/selection/selectors.ts

```diff
@@ -0,0 +1,184 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Component,
+  Input,
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  OnDestroy,
+  ViewEncapsulation,
+} from '@angular/core';
+import { DtCheckboxChange } from '@dynatrace/barista-components/checkbox';
+import { DtTable } from '../table';
+import { DtTableSelection } from './selection';
+import { Subject, Subscription } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { DtTableDataSource } from '../table-data-source';
+
+/** Handles row selection in combination with the DtTableSelection directive on the table */
+@Component({
+  selector: 'dt-table-row-selector',
+  template: ` <dt-cell class="dt-selectable-cell">
+    <dt-checkbox
+      [disabled]="_disabled(row) || _disabledDueToLimit"
+      [value]="row"
+      [checked]="_isSelected"
+      (change)="_toggleRow($event)"
+      [aria-label]="ariaLabel"
+    ></dt-checkbox>
+  </dt-cell>`,
+  styleUrls: ['./selectors.scss'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DtTableRowSelector<T> implements OnDestroy {
+  /** The rows data used for selection */
+  @Input() row: T;
+
+  @Input() ariaLabel: '';
```

Comment:
We try to stay close to native code as long as possible, please rename the input to `aria-label` and do not assign a value. Angular will not render the attribute if it is undefined, which is more helpful than an empty aria-label string. 
```
  /** Aria label of the row selector checkbox. */
  @Input('aria-label') ariaLabel: string;
```